plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/sessionconfig/CodeScanSessionConfig.kt

```
@@ -202,9 +218,11 @@ class CodeScanSessionConfig(
         val maxCountLanguage = languageCounts.filter { it.value == maxCount }.keys.firstOrNull()
 
         if (maxCountLanguage == null) {
+            programmingLanguage = CodeWhispererUnknownLanguage.INSTANCE
```

Should we throw an error here?

---

Done